openhtf/plugs/__init__.py

```diff
@@ -193,6 +194,7 @@ def __init__(self, host, port, plug_name, session):
     # Grab exactly the attrs we want, the server has all plugs', not just ours.
     for method in self.system.listMethods():
       name, attr = method.rsplit('.', 1)
+      name = name.split('.', 1)[1]
```

Comment:
so method here is also `plugs.<plug_class>.<attr>`? Maybe a function that parses this into a namedtuple that's called here and below on line 251 would be the better option?